content/apps/production-ready.md

```diff
@@ -2,54 +2,83 @@
 date: 2015-08-28T10:32:59-04:00
 menu:
   main:
-    parent: advanced
-title: Production Ready Checklist
+    parent: apps
+title: Production Ready Guide
 ---
 
-## Zero-Downtime deploy
+# Your guide to having production ready apps on cloud.gov
 
-Your application should be able to be deployed without generating any downtime.
-Check out the [cf-blue-green](https://github.com/18F/cf-blue-green) script as a
-sample script.
+Read this guide early and often, especially when you’re starting to consider a future project. It explains simple things you can do for reliable and responsive applications deployed on cloud.gov.
```

Comment:
A suggestion from a reader-centered perspective: I'd remove "simple" here. I've learned to avoid saying things are "simple" or "easy" or "just do x" in documentation, since some readers then look at the page and think "uhhh these things don't look simple to me" and unnecessarily feel concerned about their level of knowledge. :)